lib/hmmlearn/vhmm.py

```diff
@@ -784,7 +787,7 @@ def _compute_lower_bound(self, log_prob):
         if self.covariance_type != "full":
             scale_posterior_ = fill_covars(self.scale_posterior_,
                     self.covariance_type, self.n_components, self.n_features)
-            scale_prior_ = fill_covars(self.scale_posterior_,
+            scale_prior_ = fill_covars(self.scale_prior_,
```

Comment:
Is this effectively the only material change?  (I trust you on the formulas...)

Reply:
I have a follow-on MR that will then reuse most of this in a GMM, and I've been comparing the formulas across a few different papers.